spec/features/dams_collections_spec.rb

```diff
@@ -182,6 +182,11 @@
     expect(page).to have_selector("#collections-image img")
     expect(find("#collections-image img")['alt']).to eq('Heavy Metals in the Ocean Insect, Halobates')
   end
+  
+  scenario 'PAGE SHOULD HAVE COLLECTION IMAGE TITLE ATTRIBUTE IN DESKTOP VIEW' do
```

Comment:
I was going to ask why the scenario title was in all caps, and it seems the other feature tests in this block all have this. oh well.. legacy i suppose :)

Reply:
@mcritchlow - I didn't know why either.  I just copied from the previous test in this block and the title was in all caps.  I could change it if you want.  Thanks

Reply:
@mcritchlow - Scenario title has been updated.